app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        person_list.append(Person(person['name'], person['age']))
+    for key in Person.people.keys():
+        for person in people:
+            if key == person['name']:
+                if 'wife' in person and person['wife'] is not None:
+                    Person.people[key].wife = Person.people[person['wife']]
+                elif 'husband' in person and person['husband'] is not None:
+                    Person.people[key].husband \
+                        = Person.people[person['husband']]
+                else:
+                    pass
+            else:
+                pass
```

Comment:
You dont need 2 for. You can use just 
`for person in people:`
delete $16 and change key to person["name"]
or for more readability you can rename key for something more appropriate and use
key = person["name"]
Also you don't need $22-$23 because else it's not necessary

Reply:
Ok